spec/rhyme_spec.rb

```diff
@@ -4,38 +4,60 @@
 describe JacksHouse do
   subject(:jack) { JacksHouse.new }
 
-  it "generates a normal nursery rhyme" do
-    rhyme = File.read("rhyme.txt")
-    expect(jack.normal_rhyme).to eq(rhyme)
+  describe "normal nursery rhyme method" do
```

Comment:
suggestion: Instead of a textual description, the typical describe comment references the method name itself: e.g. "#normal_rhyme".

Reply:
Changed them!